files/debian-initd

```diff
@@ -206,41 +208,55 @@ case "$command" in
 		case "$?" in
 			0|1)
 				do_start
-				log_end_msg_wrapper $? 1 "always"
+				log_end_msg_wrapper "$?" 1 "always"
 				;;
 			*)
 				# Failed to stop
 				log_end_msg 1
 				;;
- 		esac
+		esac
 		;;
 
-	reload|force-reload)
-        log_daemon_msg "Reloading $DESC" "$NAME"
-        do_reload
-        log_end_msg $?
-        ;;
+	reload)
+		log_daemon_msg "Reloading $DESC" "$NAME"
+		do_reload
+		log_end_msg $?
+		;;
 
 	status)
 		log_daemon_msg "Status of $DESC"
 		do_status
 		case $? in
-			0)  log_success_msg " $NAME is running" ;;
+			0)
+				log_success_msg " $NAME is running"
+				;;
 			255)
 				check_socket
 				case $? in
-					1)  log_failure_msg " $NAME is not running" && exit 3 ;;
-					0)  log_failure_msg " $NAME is not running but $SOCKFILE exists" && exit 3 ;;
-					2)  log_failure_msg " $SOCKFILE not readable, status of $NAME is unknown" && exit 3 ;;
-					3)  log_failure_msg " $SOCKFILE exists but not a socket, status of $NAME is unknown" && exit 3 ;;
-					*)  report_bug "Unknown return code from $NAME:check_socket." && exit 4 ;;
+					1)
+						log_failure_msg " $NAME is not running" && exit 3
+						;;
+					0)
+						log_failure_msg " $NAME is not running but $SOCKFILE exists" && exit 3
+						;;
+					2)
+						log_failure_msg " $SOCKFILE not readable, status of $NAME is unknown" && exit 3
+						;;
+					3)
+						log_failure_msg " $SOCKFILE exists but not a socket, status of $NAME is unknown" && exit 3
+						;;
+					*)
+						report_bug "Unknown return code from $NAME:check_socket." && exit 4
+						;;
```

Comment:
I definitely like compressed, not blown up version more here -- easier to do visual matching of messages, exit codes etc

Reply:
Okay.

I code explicitly and in a uniform fashion, learned from bash(1)

Reply:
```
$> man 1 bash | grep uniform; echo $?
1
```
;)